tests/devapps/WAM/NetFrameworkWam/NetDesktopWinFormsWAM.csproj

```diff
@@ -14,7 +14,7 @@
     <Deterministic>true</Deterministic>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
Why did you have to do this?

Reply:
Even running on a 64 bit machine, sample app tries to load msalruntime_x86 and is not able to find it. Running dev apps on 64bit instead of any CPU